src/test/java/com/google/javascript/clutz/ClutzErrorManagerTest.java

```diff
@@ -19,7 +19,7 @@ public void testDuplicateSymbol() {
     // Useful
     assertThatProgram("/** @type {number} */ var x = 1;", "/** @type {number} */ var x = 2;")
         .diagnosticStream()
-        .containsMatch("ERROR.*Variable x first declared in");
+        .containsMatch("ERROR.*Variable x declared more than once");
```

Comment:
snapshot is broken for other reasons. Can't change it to work for both (golden tests).